product_hunt_utils.py

```diff
@@ -67,18 +115,152 @@ def get_commentor_info_for_product_id(product_id: int)-> list:
+        'cursor': None,
+        'query': '',
+    },
+    'query': 'query ProfileFollowingQuery($username:String!$cursor:String){profile:user(username:$username){id isTrashed followingsCount following(first:20 after:$cursor){edges{node{id ...UserItemFragment __typename}__typename}pageInfo{endCursor hasNextPage __typename}__typename}...MetaTags __typename}}fragment MetaTags on SEOInterface{id meta{canonicalUrl creator description image mobileAppUrl oembedUrl robots title type author authorUrl __typename}__typename}fragment UserItemFragment on User{id name headline username followersCount ...UserImage ...UserFollowButtonFragment __typename}fragment UserFollowButtonFragment on User{id followersCount isFollowed __typename}fragment UserImage on User{id name username avatarUrl __typename}',
+}
+
+    first_response = requests.post(PRODUCT_HUNT_GRAPHQL_API, cookies=cookies, headers=headers, json=json_data)
+
+    first_response = first_response.json()
+
+    id_lst = []
+
+    first_followings = first_response["data"]["profile"]["following"]["edges"]
+    end_cursor = first_response["data"]["profile"]["following"]["pageInfo"]["endCursor"]
+    next_page = first_response["data"]["profile"]["following"]["pageInfo"]["hasNextPage"]
+
+    for nodes in first_followings:
+            id_lst.append(nodes["node"]["id"])
+
+    while(next_page == True):
+        json_data = {
+        'operationName': 'ProfileFollowingQuery',
+        'variables': {
+            'username': 'alicea_hughes',
+            'cursor': end_cursor,
+            'query': '',
+        },
+        'query': 'query ProfileFollowingQuery($username:String!$cursor:String){profile:user(username:$username){id isTrashed followingsCount following(first:20 after:$cursor){edges{node{id ...UserItemFragment __typename}__typename}pageInfo{endCursor hasNextPage __typename}__typename}...MetaTags __typename}}fragment MetaTags on SEOInterface{id meta{canonicalUrl creator description image mobileAppUrl oembedUrl robots title type author authorUrl __typename}__typename}fragment UserItemFragment on User{id name headline username followersCount ...UserImage ...UserFollowButtonFragment __typename}fragment UserFollowButtonFragment on User{id followersCount isFollowed __typename}fragment UserImage on User{id name username avatarUrl __typename}',
+    }
+
+
+        response = requests.post(PRODUCT_HUNT_GRAPHQL_API, cookies=cookies, headers=headers, json=json_data)
+
+        response = response.json()
+
+        followings = response["data"]["profile"]["following"]["edges"]
+
+        end_cursor = response["data"]["profile"]["following"]["pageInfo"]["endCursor"]
+
+        next_page = response["data"]["profile"]["following"]["pageInfo"]["hasNextPage"]
+
+        for nodes in followings:
+            id_lst.append(nodes["node"]["id"])
+
+    df = pd.DataFrame(id_lst, columns = ["person id"])
+
+    df.to_csv(followingsFilePath, index = False)
+
+    df = pd.read_csv(followingsFilePath)

```

Comment:
this should be in part of `follow_now` because we are repeating the same thing `follow_persons_followers` as well. You should pass the filepath to `follow_now`